airflow-core/src/airflow/ui/src/i18n/config.ts

```diff
@@ -36,6 +38,7 @@ export const supportedLanguages = [
   { code: "de", name: "Deutsch" },
   { code: "en", name: "English" },
   { code: "ko", name: "한국어" },
+  { code: "nl", name: "Nederlands" },
```

Comment:
Mhm is this correct? Thisis not the "country code" but the name of the language. Does it need to be "Dutch"?

Reply:
@jscheffl If all the names should be in English, then yes. However, to me it seems like the name in here should be the in represented language.

Reply:
It looks correct to me